api/data_refinery_api/middleware.py

```diff
@@ -30,3 +30,17 @@ def process_response(self, request, response):
         result = client.captureMessage(message=message, data=data)
 
         return response
+
+    def process_exception(self, request, exception):
+        from raven.contrib.django.models import client
+
+        if not client.is_enabled():
+            return
```

Comment:
From https://docs.djangoproject.com/en/2.0/topics/http/middleware/#process-exception:

> process_exception() should return either None or an HttpResponse object. If it returns an HttpResponse object, the template response and response middleware will be applied and the resulting response returned to the browser. Otherwise, default exception handling kicks in.